Lib/email/message.py

```diff
@@ -101,6 +100,35 @@ def _unquotevalue(value):
         return utils.unquote(value)
 
 
+def _decode_uu(encoded):
```

Comment:
huh,, so uu library is gone but the decoding feature is still in the standard library

Reply:
Yeah I just copied this off the 3.13 stdlib, seems like a weird patch.